code-scanning/properties/sobelow.properties.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Sobelow",
+    "creator": "nccgroup",
+    "description": "Sobelow is a security-focused static analysis tool for the Phoenix framework.",
+    "iconName": "sobelow",
+    "categories": [
+      "Code Scanning",
+      "Elixir"
```

Comment:
Do you want to add `Phoenix` here as well? If there are files specific to this framework, we can enhance our starter workflows recommendation logic by detecting such files.

Reply:
If yes, please create an issue here using "Support new tech_stack" issue template.

Reply:
`Phoenix` wasn't listed in [linguist](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) so I didn't add it as a category.